src/quic/frame_reader.d

```diff
@@ -83,6 +84,11 @@ struct QuicReader(FrameType)
                 return networkStream[bufIndex-len..bufIndex];
             }
 
+            else static if (isIntegral!FieldType)
+            {
+                return readBigEndianField(networkStream, bufIndex, FieldType.sizeof);
```

Comment:
It is the expected behaviour. Useful in tls framing, where there are no VarInts.